examples/Consumer/Program.cs

```diff
@@ -124,28 +123,48 @@ public static void Run_Poll(string brokerList, List<string> topics)
         /// </summary>
         public static void Run_Consume(string brokerList, List<string> topics)
         {
-            using (var consumer = new Consumer<Ignore, string>(constructConfig(brokerList, false), null, new StringDeserializer(Encoding.UTF8)))
+            var config = new Dictionary<string, object>
+            {
+                { "bootstrap.servers", brokerList },
+                { "group.id", "csharp-consumer" },
+                { "enable.auto.commit", false },
+                { "statistics.interval.ms", 60000 },
+                { "session.timeout.ms", 6000 },
+                { "auto.offset.reset", "smallest" }
+            };
+
+            using (var consumer = new Consumer<Ignore, string>(config, null, new StringDeserializer(Encoding.UTF8)))
             {
                 // Note: All event handlers are called on the main thread.
 
                 consumer.OnPartitionEOF += (_, end)
                     => Console.WriteLine($"Reached end of topic {end.Topic} partition {end.Partition}, next message will be at offset {end.Offset}");
 
+                // Raised on critical errors, e.g. connection failures or all brokers down.
```

Comment:
connection failures are not necessarily critical failures though

Reply:
I seem to recall getting this line from you ...

Reply:
(though more likely, I probably miss-understood)